core/capabilities/webapi/outgoing_connector_handler.go

```diff
@@ -125,16 +127,28 @@ func (c *OutgoingConnectorHandler) HandleGatewayMessage(ctx context.Context, gat
 }
 
 func (c *OutgoingConnectorHandler) Start(ctx context.Context) error {
-	return c.gc.AddHandler([]string{c.method}, c)
+	return c.StartOnce("OutgoingConnectorHandler", func() error {
+		return c.gc.AddHandler([]string{c.method}, c)
+	})
 }
 
 func (c *OutgoingConnectorHandler) Close() error {
-	return nil
+	return c.StopOnce("OutgoingConnectorHandler", func() error {
+		return nil
+	})
+}
+
+func (c *OutgoingConnectorHandler) HealthReport() map[string]error {
+	return map[string]error{c.Name(): nil}
```

Comment:
`StateMachine` implements `Healthy()`:
```suggestion
	return map[string]error{c.Name(): c.Healthy()}
```